# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/interfaces/base/nsIDOMWindowUtils.idl
# Commit: 2e365eaea75b
# Full Hash: 2e365eaea75b925b2b5ff621634ba8f7d7fec00b
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-22 04:04:03
# Regressor Bug: 1568950
# File Overlap Count: 1
# Description:
#   Bug 1568950 - Part 5: Remove nsIDOMWindowUtils::forceUseCounterFlush; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D42818
# ==============================================================================

diff -r 1c1c4ca0bc14 -r 2e365eaea75b dom/interfaces/base/nsIDOMWindowUtils.idl
--- a/dom/interfaces/base/nsIDOMWindowUtils.idl	Wed Aug 21 10:29:14 2019 +0000
+++ b/dom/interfaces/base/nsIDOMWindowUtils.idl	Wed Aug 21 12:52:19 2019 +0000
@@ -1836,17 +1836,6 @@
    */
   bool hasRuleProcessorUsedByMultipleStyleSets(in unsigned long aSheetType);
 
-  /*
-   * Force the use counters for the node's associated document(s) to be
-   * flushed to telemetry.  For example, a document node will flush its own
-   * counters and an image node with an SVG source will flush the SVG
-   * document's counters.  Normally, use counters are flushed to telemetry
-   * upon document destruction, but as document destruction is somewhat
-   * non-deterministic, we have this method here for more determinism when
-   * running tests.
-   */
-  void forceUseCounterFlush(in Node aNode);
-
   /**
    * Enable or disable displayport suppression. This is intended to be used by
    * testing code, to provide more deterministic behaviour over the displayport